ng/js/app.module.js

```diff
@@ -13,6 +13,8 @@ window.boostrapjs = require('bootstrap-sass');
 window.angular = require('angular');
 window._ = require('lodash');
 require('angular-sails');
+require('angular-validator');
+require('angular-filter');
```

Comment:
Those will be removed in my next PR. Didn't end up using, forgot to remove from here and package.json too.